### PR TITLE
Support to run multiple environment at the same time

### LINKED ISSUE #24: run multiple environments at the same time
When I try to run the cli with using a file that has multiple environments, I must make a call for each one, when I really want to be able to run all of them

By adding an "all" flag to the "start" command, you could make it easier to run all environments at the same time.
example: 

mockoon-cli start --data ~/export-data.json --all

I can contribute with this feature :)

### PR DESCRIPTION
**Pull request by Eitol**
closes #24

**Parent issue** 

Closes #24

### CODE REVIEW

File: src/commands/start.ts [hunk 1]
```diff
@@ -52,115 +59,172 @@ export default class Start extends Command {
...
+
+  private async runEnvironment(environmentInfo: EnvironmentInfo) {
+    const process: Proc = await this.startProcess(environmentInfo);
+
+    if (process[0].pm2_env.status === 'errored') {
+      // if process is errored we want to delete it
+      await this.handleProcessError(environmentInfo.name);
+    }
 
+    await this.logStartedProcess(environmentInfo, process);
```
Comment (255kb) [inline src/commands/start.ts:89]: You can remove the `await`, function is not returning a promise.

File: src/commands/start.ts [hunk 2]
```diff
@@ -52,115 +59,172 @@ export default class Start extends Command {
...
+
+  private async addProcessToProcessListManager(environmentInfo: EnvironmentInfo, process: Proc) {
+    await ProcessListManager.addProcess({
+      name: environmentInfo.name,
+      port: environmentInfo.port,
+      pid: process[0].pm2_env.pm_id
+    });
+  }
+
+  private async logStartedProcess(environmentInfo: EnvironmentInfo, process: Proc) {
```
Comment (255kb) [inline src/commands/start.ts:102]: You can remove the `async`, function is not returning a promise.

**Review by 255kb — CHANGES_REQUESTED** [review]
Thank you for your contribution. 
In addition to the comments I made, could you please apply Prettier on all the modified files? 

Also, concerning the implementation, when you try to run multiple mocks while one port is in use (e.g. 3000/3001/3002, but 3000 is in use) it will error and exit immediately. I think we could use `this.error(..., {exit: false})` when running multiple mocks. So, we can still have two mocks running. What do you think? 

Of course this would require some changes as `this.error` won't throw anymore.

**Comment by Eitol** [comment]
> Also, concerning the implementation, when you try to run multiple mocks while one port is in use (e.g. 3000/3001/3002, but 3000 is in use) it will error and exit immediately. I think we could use `this.error(..., {exit: false})` when running multiple mocks. So, we can still have two mocks running. What do you think?
> 
> Of course this would require some changes as `this.error` won't throw anymore.

RE: Yes, maybe adding an additional flag called "skip-fails"
But maybe it is another additional feature

Thank you for accepting contributions from the community. Your tool has helped us a lot!

**Review by 255kb — APPROVED** [review]

**Comment by 255kb** [comment]
> RE: Yes, maybe adding an additional flag called "skip-fails"
> But maybe it is another additional feature

You are that could be an improvement in the future. 
Thank you for updating your contribution and adding the documentation!

**Pull request merged**